demo/src/store/initialState.js

```diff
@@ -0,0 +1,353 @@
+import bigYawnThumbnail from '../images/thumbnails/big_yawn.png';
+import caughtThumbnail from '../images/thumbnails/caught.png';
+import curledFeetsiesThumbnail from '../images/thumbnails/curled_feetsies.png';
+import downwardFacingCatThumbnail from '../images/thumbnails/downward_facing_cat.png';
+import eyeBoogiesThumbnail from '../images/thumbnails/eye_boogies.png';
+import fluffyThumbnail from '../images/thumbnails/fluffy.png';
+import handsomeCatThumbnail from '../images/thumbnails/handsome_cat.png';
+import hardToCodeWithThumbnail from '../images/thumbnails/hard_to_code_with.png';
+import kittenThumbnail from '../images/thumbnails/kitten.png';
+import lightOfMyLifeThumbnail from '../images/thumbnails/light_of_my_life.png';
+import litterNoseThumbnail from '../images/thumbnails/litter_nose.png';
+import makinBiscuitsThumbnail from '../images/thumbnails/makin_biscuits.png';
+import mirinThumbnail from '../images/thumbnails/mirin.png';
+import pawsUpThumbnail from '../images/thumbnails/paws_up.png';
+import petMyBellyThumbnail from '../images/thumbnails/pet_my_belly.png';
+import phoneBackgroundThumbnail from '../images/thumbnails/phone_background.png';
+import pinkToeBeanThumbnail from '../images/thumbnails/pink_toe_bean.png';
+import shinyThumbnail from '../images/thumbnails/shiny.png';
+import snoozingThumbnail from '../images/thumbnails/snoozing.png';
+import snoringThumbnail from '../images/thumbnails/snoring.png';
+import snuggledThumbnail from '../images/thumbnails/snuggled.png';
+import solemnThumbnail from '../images/thumbnails/solemn.png';
+import spoiledThumbnail from '../images/thumbnails/spoiled.png';
+import stopThumbnail from '../images/thumbnails/stop.png';
+import tinyBlepThumbnail from '../images/thumbnails/tiny_blep.png';
+import tuckedInKittyThumbnail from '../images/thumbnails/tucked_in_kitty.png';
+import whatYouLookinAtThumbnail from '../images/thumbnails/what_you_lookin_at.png';
+import wheresPoeThumbnail from '../images/thumbnails/wheres_poe.png';
+
+
+/**
+ * The items in the store; each has name, image, cost, and
+ * description. Each item is stored along with the quantity in the user's cart
+ * and if it is in the user's cart. If the item is not in the cart,
+ * the quantity is 0, but the converse is not always true: users that modify
+ * how many items are in their cart will usually not want them to
+ * disappear immediately.
+ *
+ * When adding a new item, generate a [v4 UUID](https://www.uuidgenerator.net/version4)
+ * and copy the first five characters to serve as the unique item ID.
+ * @typedef {!Object<string,
+  *      {name:string, image:!Object, cost:number, quantity:number,
+  *      inCart:boolean, description:string}>} ItemStore
+  */
+export const initialState = {
+  items: {
+    '1jef2': {
+      name: 'Airplane Ears',
+      image: airplaneEars,
+      thumbnail: airplaneEarsThumbnail,
+      cost: 25,
+      quantity: 1,
+      inCart: true,
```

Comment:
Is it intended (e.g. for testing) that some items start with a non-zero quantity and/or in the cart?

Intuitively it seems like the cart should start empty, i.e. all items with quantity 0 and inCart false.

Reply:
Yes, it is purely for testing purposes. Once we have the add to cart mechanic merged we'll remove it.